app/src/main/java/org/oppia/android/app/recyclerview/BindableAdapter.kt

```diff
@@ -242,10 +195,17 @@ class BindableAdapter<T : Any> internal constructor(
       )
     }
 
-    companion object {
-      /** Returns a new [SingleTypeBuilder]. */
-      inline fun <reified T : Any> newBuilder(): SingleTypeBuilder<T> {
-        return SingleTypeBuilder(T::class)
+    /**
+     * An injectable factory to create [SingleTypeBuilder]s.
+     *
+     * Note that singleTypeBuilders can only be used in fragment-bounded classes.
```

Comment:
```suggestion
     * Note that this class can only be used in fragment-bounded classes.
```

Small nit--ditto below.

Reply:
Done.